src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -165,7 +165,7 @@ class SignInModal extends React.Component {
             <div className="help-info">
               <h4>Having trouble signing in?</h4>
               <p>
-                <a href="/faq/" target="_blank">
+                <a href="/sign-in-faq/" target="_blank">
```

Comment:
Unless we're okay with the outdated links on Vets.gov, we'll have to put this behind a feature flag to make sure we render the right link on each site. Something like -

```jsx
import brandConsolidation from '../../../brand-consolidation';

const faqHref = brandConsolidation.isEnabled() ? '/sign-in-faq/' : '/faq/';

// ....
 <a href={faqHref} target="_blank">

```

Reply:
Thanks guys...updated in https://github.com/department-of-veterans-affairs/vets-website/pull/8833/commits/6add2497778d08f3ebf7d689d72c7407249b4a5a